modules/claims_api/lib/claims_api/v2/params_validation/intent_to_file/intent_to_file_validator.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    module ParamsValidation
+      module IntentToFile
+        class IntentToFileValidator < ActiveModel::Validator
+          def validate(record)
```

Comment:
I think we're gonna want to also add validation logic around burial ITFs like [this](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb#L110-L124).

Reply:
Ah, if you check the AC on the ticket, you'll see that it was decided that we're not going to support `burial` ITF's in V2 because we don't have the capability of supporting Claimants who aren't Veterans.
> AC:

Allow submission of the following types of intent to files
compensation
pension
NOT burial (can't do this without supporting claimants who aren't Veterans)

Reply:
OPE